project-koku/koku
# COST-376: Fix bug order by delta during pagination.

Jira Issue: https://issues.redhat.com/browse/COST-376

This pr is to fix a bug that bubbled up during my work on adding default pagination. I noticed that once I added the default pagination that the `order_by[delta]` tests were failing with a 500 error. 

#### Replicate bug on master

Hit these endpoints combinations cause a 500 fail before customer data is added and after the customer data is present. 

- http://127.0.0.1:8000/api/cost-management/v1/reports/aws/instance-types/?order_by[delta]=asc
- http://127.0.0.1:8000/api/cost-management/v1/reports/aws/instance-types/?filter[limit]=5&filter[offset]=0&order_by[delta]=asc

---

#### How to test

Pull down this branch and bring up the server. 
Before loading customer data hit these endpoints:

- http://127.0.0.1:8000/api/cost-management/v1/reports/aws/instance-types/?order_by[delta]=asc
- http://127.0.0.1:8000/api/cost-management/v1/reports/aws/instance-types/?filter[limit]=5&filter[offset]=0&order_by[delta]=asc

Notice that they no longer fail.

Then add some customer data and hit the same points again to notice that they no longer 500 & order by the delta.
